app/src/main/res/values/donottranslate_questionnaire_values.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Just curious, how do you create `string-array` with non translate words? I was thinking `donottranslate.xml` is the best way to do it.